lib/rubocop/cop/rails/action_controller_flash_before_render.rb

```diff
@@ -69,8 +69,8 @@ def on_send(flash_node)
         def followed_by_render?(flash_node)
           flash_assigment_node = find_ancestor(flash_node, type: :send)
           context = flash_assigment_node
-          if (if_node = context.each_ancestor(:if).first)
-            context = if_node
+          if (node = context.each_ancestor(:if, :rescue).first)
+            context = node
```

Comment:
Oops. CI has been failed:

```
Offenses:

lib/rubocop/cop/rails/action_controller_flash_before_render.rb:79:36: W: Lint/ShadowingOuterLocalVariable: Shadowing outer local variable - node.
          context.compact.any? do |node|
                                   ^^^^

277 files inspected, 1 offense detected
RuboCop failed!
```

https://app.circleci.com/pipelines/github/rubocop/rubocop-rails/2201/workflows/98db9a47-7ef0-404b-9248-d91268dbf252/jobs/14728

Can you update with the below and squash this PR's commits into one?

```diff
% g diff .
diff --git a/lib/rubocop/cop/rails/action_controller_flash_before_render.rb b/lib/rubocop/cop/rails/action_controller_flash_before_render.rb
index 08419f098..4a30ac97e 100644
--- a/lib/rubocop/cop/rails/action_controller_flash_before_render.rb
+++ b/lib/rubocop/cop/rails/action_controller_flash_before_render.rb
@@ -76,8 +76,8 @@ module RuboCop
           end
           context = context.right_siblings

-          context.compact.any? do |node|
-            render?(node)
+          context.compact.any? do |render_candidate|
+            render?(render_candidate)
           end
         end
```

Reply:
Sorry for my delay. It's now updated and squashed.